AK/Atomic.h

```diff
@@ -10,6 +10,10 @@
 #include <AK/Platform.h>
 #include <AK/Types.h>
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wpragmas"
+#pragma GCC diagnostic ignored "-Watomic-alignment"
```

Comment:
I feel like this should have a comment

Reply:
Ah yeah. This one is something I saw when (intentionally or not) targeting i686 and/or armv7 with the NDK toolchain.

e.g. https://github.com/llvm/llvm-project/issues/37941

Honestly, I don't think anyone will try using Lagom with those architectures anytime soon, so I'll remove this from the patch.